core/storage/user/gae_models.py

```diff
@@ -1629,3 +1629,37 @@ def increment_score_for_user(cls, user_id, score_category, increment_by):
         else:
             model.score += increment_by
             model.put()
+
+
+class UserCommunityRightsModel(base_models.BaseModel):
+    """Model for storing user's rights on community dashboard.
+
+    Instances of this class are keyed by the user id.
+    """
+    # User id used to identify user by GAE. Is not required for now because we
+    # need to perform migration to fill this for existing users.
+    gae_id = ndb.StringProperty(required=False, indexed=True)
```

Comment:
Wait, this should just use user_id, right, without referencing gae_id? /cc @vojtechjelinek

Reply:
Yeah, just use user id not gae id.